specification/vmware/Microsoft.AVS/routes.tsp

```diff
@@ -658,7 +709,10 @@ interface WorkloadNetworkDnsServices {
 interface WorkloadNetworkDnsZones {
   #suppress "@azure-tools/typespec-azure-core/no-operation-id" "Can not change existing operationId."
   @operationId("WorkloadNetworks_ListDnsZones")
-  list is ArmResourceListByParent<WorkloadNetworkDnsZone>;
+  list is ArmResourceListByParent<
+    WorkloadNetworkDnsZone,
+    Response = ArmResponse<ResourceList<WorkloadNetworkDnsZone, "{name}sList">>
```

Comment:
maybe a minor concern, is this the right way to get plural if the name is policy?